src/components/CustomPressable/index.js

```diff
@@ -19,7 +19,7 @@ const CustomPressable = (props) => {
       onPress={onPress}
       style={style}
       android_ripple={{
-        color: colors.gray1,
+        color: COLORS.gray9,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value for the Android ripple effect has been changed from `colors.gray1` to `COLORS.gray9`. Ensure that this color change is intentional and aligns with the overall theme and design guidelines of your application.
```diff
- color: colors.gray1,
+ color: COLORS.gray9,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->